src/main/java/meteordevelopment/meteorclient/systems/modules/misc/AutoLog.java

```diff
@@ -94,19 +94,20 @@ public AutoLog() {
 
     @EventHandler
     private void onTick(TickEvent.Post event) {
-        if (mc.player.getHealth() <= 0) {
+        float playerHealth = mc.player.getHealth();
+        if (playerHealth <= 0) {
```

Comment:
This ignores the `toggleOff` and `smartToggle` completely.

Reply:
you sure? all i did was store the result of `mc.player.getHealth()` to a variable and use that instead of calling the method multiple times, i didnt change the structure of the code

Reply:
Ah mb